ui/src/app/cohort-search/list-search/list-search.component.tsx

```diff
@@ -371,7 +372,16 @@ export const ListSearch = withCurrentWorkspace()(
             &nbsp;<Clickable style={styles.vocabLink}
               onClick={() => this.getResults(sourceMatch.code)}>
               Return to source code
-          </Clickable>.
+            </Clickable>.
+          </div>}
+          {isStandard !== undefined && <div style={{...styles.error, margin: '-0.5rem 0 0.5rem'}}>
+            <div style={{float: 'left', height: '1.5rem'}}>
+              <ClrIcon style={{margin: '0 0.5rem 0 0.25rem'}} className='is-solid'
+                shape='exclamation-triangle' size='22'/>
+            </div>
+            You have added a {isStandard ? 'standard' : 'source'} code. To add a
```

Comment:
this calculation is repeated. Maybe do `const codeType = isStandard ? 'standard' : 'source'` above and use that here and below.

Reply:
It looks the same but actually switches strings in the second instance.
ln 382: `isStandard ? ' standard' : 'source'` 
ln 383: `isStandard ? ' source' : ' standard'`